<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="snap_cam" type="snap_cam_publisher_node" name="snap_cam_highres_publisher" output="screen">

    <!--
    "optflow" or "highres" -->
    <param name="camera_type" value="highres"/>

    <!--
    highres:      QVGA  : (320,240)
                  VGA   : (640,480)
                  720p  : (1280,720)
                  1080p : (1920,1080)
                  4k    : (4096,2160) -->
    <param name="camera_resolution" value="4k"/>

    <!--
    highres: 0: 15 1: 24 2: 30 3: 60 4: 90 6: 120 -->
    <param name="camera_fps_idx" value="0"/>

    <param name="topic_name" value="image"/>
  </node>
</launch>
